sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -477,6 +477,7 @@ def _to_rest_object(self) -> JobBase:
             inputs=to_rest_dataset_literal_inputs(built_inputs, job_type=self.type),
             outputs=to_rest_data_outputs(built_outputs),
             settings=settings_dict,
+            services={k: v._to_rest_object() for k, v in self.services.items()} if self.services else None,
```

Comment:
Please add a ut to test the lines added in this PR. Can add in test_pipeline_job_entity.py to test pipeline job to rest obj and rest obj to pipeline job

Reply:
@TonyJ1 will add test in next PR